gradle/extensions/doclib-resource-command-override/src/main/java/com/liferay/blade/samples/doclib/resourcecommand/override/CustomMVCResourceCommand.java

```diff
@@ -60,6 +61,11 @@ public boolean serveResource(
 		return true;
 	}
 
+	public void showLog() {
```

Comment:
No method calls this, so we need to add the log to another method that is called and possibly in the activate.